x/reward/client/cli/query.go

```diff
@@ -32,23 +34,33 @@ func GetQueryCmd() *cobra.Command {
 func GetCmdInflationRate() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "inflation-rate",
-		Short: "Returns the inflation rate on the network",
+		Short: "Returns the inflation rate on the network. If a validator is provided, query the inflation rate for that validator.",
 		Args:  cobra.ExactArgs(0),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientQueryContext(cmd)
-			if err != nil {
-				return err
-			}
+	}
 
-			queryClient := types.NewQueryServiceClient(clientCtx)
+	validator := cmd.Flags().String("validator", "", "the validator to retrieve the inflation rate for")
```

Comment:
why is this a flag instead of an optional argument?

Reply:
we've been using flags for all other cli cmds so far